lukasz-lapczyk/review-dealer
# Add Google authentication

# Overview
Users can now sign up to the site using their Google accounts.

Environment variables:
GOOGLE_CLIENT_ID=208574874516-bmofk164qs3dmlpaqj27l5h7o8bj514e.apps.googleusercontent.com
GOOGLE_CLIENT_SECRET=JEVgZLsCv4K3n-LDndcEPsEZ

# Testing prerequisites
- bundle install

# Testing 
- navigate to root directory
- click 'Sign up' link on navbar
- click 'Sign in with GoogleOauth2' link below the sign up form
- [x] you should be logged in with your Google account
